tests/rptest/services/redpanda_cloud.py

```diff
@@ -101,39 +134,40 @@ def partitions_upper_limit(self):
     #    |        |        |   |         |          |   +- partitions_max
     #    |        |        |   |         |          |   |      +- connections_limit
     #    |        |        |   |         |          |   |      |     +- memory_per_broker
-    #    |        |        |   |         |          |   |      |     |
+    #    |        |        |   |         |          |   |      |     |       +- machine_type
+    #    |        |        |   |         |          |   |      |     |       |
     CloudTierName.AWS_1: AdvertisedTierConfig(
-         20*MiB,  60*MiB,  3,  512*MiB,  300*GiB,   20, 1000,  1500, 32*GiB
+         20*MiB,  60*MiB,  3,  512*MiB,  300*GiB,   20, 1000,  1500, 32*GiB, MachineTypeName.I3EN_XLARGE,
```

Comment:
Not part of this change but these should all be MB and GB (10^6 and 10^9) except for memory.